### PR TITLE
implement LinuxTerminalOpener

### LINKED ISSUE #483: Implement LinuxTerminalOpener

### PR DESCRIPTION
**Pull request by plata**
fixes #483

### CODE REVIEW

File: phoenicis-tools/src/main/java/org/phoenicis/tools/system/SystemConfiguration.java [hunk 1]
```diff
@@ -21,14 +21,21 @@
...
+
 @Configuration
 public class SystemConfiguration {
+    @Value("${tools.terminal}")
+    private String terminalCommand;
+
     @Bean
     public TerminalOpener terminalOpener() {
-        return new AutomaticTerminalOpener(terminalOpenerFactory(), operatingSystemFetcher());
+        return new AutomaticTerminalOpener(terminalOpenerFactory(), operatingSystemFetcher(),
```
Comment (qparis) [inline phoenicis-tools/src/main/java/org/phoenicis/tools/system/SystemConfiguration.java]: The name of the command is specific to Linux so it should be passed to the constructor instead

Comment (plata) [inline phoenicis-tools/src/main/java/org/phoenicis/tools/system/SystemConfiguration.java]: I'm not sure I understand you correctly. Could you be a bit more precise on how it should look like?

**Comment by madoar** [comment]
@qparis I think you should approve this, because I don't really know how to evaluate this.

**Comment by madoar** [comment]
@plata this PR seems to have conflicts.

**Comment by plata** [comment]
@qparis please check if it's ok now.

**Comment by madoar** [comment]
@plata it seems we have a conflict in this PR

**Comment by madoar** [comment]
@qparis I think this PR is ready to be merged, what do you think?

**Pull request merged**